src/modules/paste.ts

```diff
@@ -109,10 +110,24 @@ export function paste(editor: Editor, options?: PasteOptions) {
         editor.delete([ at, to ]);
       }
     }
+  }
 
+  function onPaste(event: ClipboardEvent) {
+    if (!editor.enabled || !editor.doc.selection) return;
+    event.preventDefault();
+    const dataTransfer = event.clipboardData;
+    const { doc } = editor;
+    const selection = doc.selection && normalizeRange(doc.selection);
+    if (!dataTransfer || !selection) return;
```

Comment:
We could remove line 120 and just check for `doc.selection` here. There is no need to normalize the range (sort the lower number first) when it is not being used anywhere other than to check if it is not null.

Reply:
updated.